bin/postgres-ha/modules/pgmonitor.sh

```diff
@@ -47,18 +47,11 @@ then
         cp "${function_file}" "/tmp/setup_pg.sql"
         sed -i "s/\/usr\/bin\/pgbackrest-info.sh/\/opt\/cpm\/bin\/pgbackrest\/pgbackrest_info.sh/g" "/tmp/setup_pg.sql"
 
-        # TODO Add ON_ERROR_STOP and single transaction when
-        # upstream pgmonitor changes setup SQL to check if the
-        # role exists prior to creating it.
         psql -U postgres --port="${PG_PRIMARY_PORT}" -d postgres \
             < "/tmp/setup_pg.sql" > /tmp/pgmonitor-setup.stdout 2> /tmp/pgmonitor-setup.stderr
 
-        #err_check "$?" "pgMonitor Setup" "Could not load pgMonitor functions: \n$(cat /tmp/pgmonitor.stderr)"
-
         psql -U postgres --port="${PG_PRIMARY_PORT}" -d postgres \
             -c "ALTER ROLE ccp_monitoring PASSWORD '${PGMONITOR_PASSWORD?}'" \
```

Comment:
It may behoove us to make this:

```suggestion
            -c "SET log_statement TO 'none'; ALTER ROLE ccp_monitoring PASSWORD '${PGMONITOR_PASSWORD?}'" \
```

since I'm confident the password is in plaintext at this point, and this only turns off the logging for the session (the `psql` call). Perhaps a friendly `echo_info` after that if the command succeeds, that the password was updated successfully.

Reply:
Command modified per recommendation.